src/views/Generator/GeneratorControls/GeneratorControls.tsx

```diff
@@ -38,52 +39,55 @@ export function GeneratorControls({ onSave, isDirty }: GeneratorControlsProps) {
 
   return (
     <>
-      <ButtonWithTooltip
-        onClick={onSave}
-        disabled={!isDirty}
-        tooltip={!isDirty ? 'Changes saved' : ''}
-      >
-        Save generator
-      </ButtonWithTooltip>
-      <DropdownMenu.Root>
-        <DropdownMenu.Trigger>
-          <IconButton variant="ghost" color="gray">
-            <DotsVerticalIcon />
-          </IconButton>
-        </DropdownMenu.Trigger>
-        <DropdownMenu.Content>
-          <DropdownMenu.Item
-            onSelect={() => setIsValidatorDialogOpen(true)}
-            disabled={!isScriptExportable}
-          >
-            Validate script
-          </DropdownMenu.Item>
-          <DropdownMenu.Item
-            onSelect={() => setIsExportScriptDialogOpen(true)}
-            disabled={!isScriptExportable}
-          >
-            Export script
-          </DropdownMenu.Item>
-          <DropdownMenu.Separator />
-          <DropdownMenu.Item onSelect={handleDeleteGenerator} color="red">
-            Delete generator
-          </DropdownMenu.Item>
-        </DropdownMenu.Content>
-      </DropdownMenu.Root>
-      {isScriptExportable && (
-        <>
-          <ValidatorDialog
-            script={preview}
-            open={isValidatorDialogOpen}
-            onOpenChange={setIsValidatorDialogOpen}
-          />
-          <ExportScriptDialog
-            onExport={exportScript}
-            open={isExportScriptDialogOpen}
-            onOpenChange={setIsExportScriptDialogOpen}
-          />
-        </>
-      )}
+      <RecordingSelector />
```

Comment:
I think the `+` button might be a bit confusing with this new placement: it's not immediately clear if it's a part of the recording selected, the save generator button, or something else entirely. Perhaps we can add a gap/margin so that the hierarchy is clearer?
<img width="535" alt="image" src="https://github.com/user-attachments/assets/8351b79e-184f-48bb-b161-9a0880333f52" />

Reply:
I considered using a vertical divider as an alternative, but I kept the extra margin so it's easier to manage in smaller resolutions.